### PR TITLE
Correct spelling errors

### PR DESCRIPTION
**Pull request by jmthomas**

### CODE REVIEW

File: openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-scriptrunner/src/tools/ScriptRunner/Dialogs/AskDialog.vue [hunk 1]
```diff
@@ -126,9 +126,9 @@ export default {
...
 </script>
 
-<sytle lang="scss" scoped>
+<style lang="scss" scoped>
 .question {
   font-size: 1rem;
   white-space: pre-line;
 }
-</sytle>
+</style>
```
Comment (jmthomas) [inline openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-scriptrunner/src/tools/ScriptRunner/Dialogs/AskDialog.vue:134]: Actual bug

File: openc3/lib/openc3/utilities/metric.rb [hunk 2]
```diff
@@ -77,7 +76,7 @@ def set(name:, value:, type: nil, unit: nil, help: nil, labels: nil, time_ms: ni
     end
 
     def set_multiple(data)
-      @mutex.synchonize do
+      @mutex.synchronize do
```
Comment (jmthomas) [inline openc3/lib/openc3/utilities/metric.rb:79]: Actual bug

File: README.md [hunk 3]
```diff
@@ -27,13 +27,13 @@ After configuring OpenC3 COSMOS to talk to your hardware, you immediately can us
...
 
 1. **Script Runner**
 
    - Script Runner allows for running OpenC3 COSMOS test procedures or any other Ruby code from a graphical environment that highlights each line as it executes. At anytime during execution, the script can be paused or stopped. If a telemetry check fails or any other exception occurs, the script is immediately stopped and the user notified.
 
-   - Script Runner also allows you to break your operational or test procedures down into discreet test cases that each complete with either SUCCESS or FAILURE. After running, a script report is automatically created for you. Convenient features such as the ability to loop testing help get the kinks out of your system before formal runs.
+   - Script Runner also allows you to break your operational or test procedures down into discret test cases that each complete with either SUCCESS or FAILURE. After running, a script report is automatically created for you. Convenient features such as the ability to loop testing help get the kinks out of your system before formal runs.
```
Comment (ryanmelt) [inline README.md:36]: should be discrete

**Review by ryanmelt — CHANGES_REQUESTED** [review]
That's a lot of spelling errors!
One spelling error still "discret" is French.
Also please add a spell checker to our CI/CD pipeline so that these don't start building up again.

**Review by ryanmelt — APPROVED** [review]

**Pull request merged**